chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
We are using @PreAuthorize on methods in the SurveillanceManager to restrict access to users for create/delete/update, for example:

@PreAuthorize("hasRole('ROLE_ADMIN') or "
			+ "((hasRole('ROLE_ACB_STAFF') or hasRole('ROLE_ACB_ADMIN')) "
			+ "and hasPermission(#acbId, 'gov.healthit.chpl.dto.CertificationBodyDTO', admin))")